tck/src/main/java/org/eclipse/microprofile/graphql/tck/apps/superhero/api/HeroFinder.java

```diff
@@ -107,12 +108,40 @@ public Team removeHeroFromTeam(@Argument("hero") String heroName,
 
     @Mutation(description="Removes a hero... permanently...")
     public Collection<SuperHero> removeHero(@Argument("hero") String heroName) throws UnknownHeroException {
+        LOG.info("removeHero invoked");
         if (heroDB.removeHero(heroName) == null) {
             throw new UnknownHeroException(heroName);
         }
         return allHeroes();
     }
 
+    @Mutation(description="Gives a hero new equipment")
+    public SuperHero provisionHero(@Argument("hero") String heroName,
```

Comment:
This might be pedantic. Should the method name not rather be addItemtoHere, or provisionItemtoHere (to be the same format as removeItemFromHere) ?